# HG changeset patch
# Parent 087083483b7b6b50467f2e0440ab8d284a12cdfb
# User Frédéric Wang <fred.wang@free.fr>
MathML Operator Dictionary: sort entries of the dictionary and remove minsize attributes (bug 534970). r=karlt

diff --git a/layout/mathml/mathfont.properties b/layout/mathml/mathfont.properties
--- a/layout/mathml/mathfont.properties
+++ b/layout/mathml/mathfont.properties
@@ -114,506 +114,508 @@ mathvariant.bold-sans-serif = A\ud835\ud
 mathvariant.sans-serif-italic = A\ud835\ude08B\ud835\ude09C\ud835\ude0aD\ud835\ude0bE\ud835\ude0cF\ud835\ude0dG\ud835\ude0eH\ud835\ude0fI\ud835\ude10J\ud835\ude11K\ud835\ude12L\ud835\ude13M\ud835\ude14N\ud835\ude15O\ud835\ude16P\ud835\ude17Q\ud835\ude18R\ud835\ude19S\ud835\ude1aT\ud835\ude1bU\ud835\ude1cV\ud835\ude1dW\ud835\ude1eX\ud835\ude1fY\ud835\ude20Z\ud835\ude21a\ud835\ude22b\ud835\ude23c\ud835\ude24d\ud835\ude25e\ud835\ude26f\ud835\ude27g\ud835\ude28h\ud835\ude29i\ud835\ude2aj\ud835\ude2bk\ud835\ude2cl\ud835\ude2dm\ud835\ude2en\ud835\ude2fo\ud835\ude30p\ud835\ude31q\ud835\ude32r\ud835\ude33s\ud835\ude34t\ud835\ude35u\ud835\ude36v\ud835\ude37w\ud835\ude38x\ud835\ude39y\ud835\ude3az\ud835\ude3b
 mathvariant.sans-serif-bold-italic = A\ud835\ude3cB\ud835\ude3dC\ud835\ude3eD\ud835\ude3fE\ud835\ude40F\ud835\ude41G\ud835\ude42H\ud835\ude43I\ud835\ude44J\ud835\ude45K\ud835\ude46L\ud835\ude47M\ud835\ude48N\ud835\ude49O\ud835\ude4aP\ud835\ude4bQ\ud835\ude4cR\ud835\ude4dS\ud835\ude4eT\ud835\ude4fU\ud835\ude50V\ud835\ude51W\ud835\ude52X\ud835\ude53Y\ud835\ude54Z\ud835\ude55a\ud835\ude56b\ud835\ude57c\ud835\ude58d\ud835\ude59e\ud835\ude5af\ud835\ude5bg\ud835\ude5ch\ud835\ude5di\ud835\ude5ej\ud835\ude5fk\ud835\ude60l\ud835\ude61m\ud835\ude62n\ud835\ude63o\ud835\ude64p\ud835\ude65q\ud835\ude66r\ud835\ude67s\ud835\ude68t\ud835\ude69u\ud835\ude6av\ud835\ude6bw\ud835\ude6cx\ud835\ude6dy\ud835\ude6ez\ud835\ude6f\u0391\ud835\udf90\u0392\ud835\udf91\u0393\ud835\udf92\u0394\ud835\udf93\u0395\ud835\udf94\u0396\ud835\udf95\u0397\ud835\udf96\u0398\ud835\udf97\u0399\ud835\udf98\u039A\ud835\udf99\u039B\ud835\udf9a\u039C\ud835\udf9b\u039D\ud835\udf9c\u039E\ud835\udf9d\u039F\ud835\udf9e\u03A0\ud835\udf9f\u03A1\ud835\udfa0\u03F4\ud835\udfa1\u03A3\ud835\udfa2\u03A4\ud835\udfa3\u03A5\ud835\udfa4\u03A6\ud835\udfa5\u03A7\ud835\udfa6\u03A8\ud835\udfa7\u03A9\ud835\udfa8\u2207\ud835\udfa9\u03B1\ud835\udfaa\u03B2\ud835\udfab\u03B3\ud835\udfac\u03B4\ud835\udfad\u03B5\ud835\udfae\u03B6\ud835\udfaf\u03B7\ud835\udfb0\u03B8\ud835\udfb1\u03B9\ud835\udfb2\u03BA\ud835\udfb3\u03BB\ud835\udfb4\u03BC\ud835\udfb5\u03BD\ud835\udfb6\u03BE\ud835\udfb7\u03BF\ud835\udfb8\u03C0\ud835\udfb9\u03C1\ud835\udfba\u03C2\ud835\udfbb\u03C3\ud835\udfbc\u03C4\ud835\udfbd\u03C5\ud835\udfbe\u03C6\ud835\udfbf\u03C7\ud835\udfc0\u03C8\ud835\udfc1\u03C9\ud835\udfc2\u2202\ud835\udfc3\u220A\ud835\udfc4\u03D1\ud835\udfc5\u03F0\ud835\udfc6\u03D5\ud835\udfc7\u03F1\ud835\udfc8\u03D6\ud835\udfc9
 mathvariant.monospace = A\ud835\ude70B\ud835\ude71C\ud835\ude72D\ud835\ude73E\ud835\ude74F\ud835\ude75G\ud835\ude76H\ud835\ude77I\ud835\ude78J\ud835\ude79K\ud835\ude7aL\ud835\ude7bM\ud835\ude7cN\ud835\ude7dO\ud835\ude7eP\ud835\ude7fQ\ud835\ude80R\ud835\ude81S\ud835\ude82T\ud835\ude83U\ud835\ude84V\ud835\ude85W\ud835\ude86X\ud835\ude87Y\ud835\ude88Z\ud835\ude89a\ud835\ude8ab\ud835\ude8bc\ud835\ude8cd\ud835\ude8de\ud835\ude8ef\ud835\ude8fg\ud835\ude90h\ud835\ude91i\ud835\ude92j\ud835\ude93k\ud835\ude94l\ud835\ude95m\ud835\ude96n\ud835\ude97o\ud835\ude98p\ud835\ude99q\ud835\ude9ar\ud835\ude9bs\ud835\ude9ct\ud835\ude9du\ud835\ude9ev\ud835\ude9fw\ud835\udea0x\ud835\udea1y\ud835\udea2z\ud835\udea30\ud835\udff61\ud835\udff72\ud835\udff83\ud835\udff94\ud835\udffa5\ud835\udffb6\ud835\udffc7\ud835\udffd8\ud835\udffe9\ud835\udfff
 
 
 # Operator Dictionary indexed on the "form" (i.e., infix, prefix, or suffix).
 # Each entry lists the attributes of the operator, using its Unicode format.
 
-operator.\u0028.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # (
-operator.\u0029.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # )
-operator.\u005B.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # [
-operator.\u005D.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # ]
-operator.\u007B.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # {
-operator.\u007D.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # }
-operator.\u201D.postfix = fence lspace:0 rspace:0 # &CloseCurlyDoubleQuote;
-operator.\u2019.postfix = fence lspace:0 rspace:0 # &CloseCurlyQuote;
-operator.\u27E8.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &LeftAngleBracket;
-operator.\u2308.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &LeftCeiling;
-operator.\u27E6.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &LeftDoubleBracket;
-operator.\u230A.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &LeftFloor;
-operator.\u201C.prefix = fence lspace:0 rspace:0 # &OpenCurlyDoubleQuote;
-operator.\u2018.prefix = fence lspace:0 rspace:0 # &OpenCurlyQuote;
-operator.\u27E9.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &RightAngleBracket;
-operator.\u2309.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &RightCeiling;
-operator.\u27E7.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &RightDoubleBracket;
-operator.\u230B.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &RightFloor;
-operator.\u2063.infix = separator lspace:0 rspace:0 # &InvisibleComma;
-operator.\u002C.infix = separator lspace:0 rspace:6 # ,
-operator.\u2500.infix = direction:horizontal stretchy minsize:0 lspace:0 rspace:0 # &HorizontalLine;
-operator.\u007C.infix = direction:vertical stretchy symmetric minsize:0 lspace:0 rspace:0 # &VerticalLine; |
-operator.\u003B.infix = separator lspace:0 rspace:5 # ;
-operator.\u003B.postfix = separator lspace:0 rspace:0 # ;
-operator.\u003A\u003D.infix = lspace:5 rspace:5 # :=
-operator.\u2254.infix = lspace:5 rspace:5 # &Assign;
-operator.\u2235.infix = lspace:5 rspace:5 # &Because;
-operator.\u2234.infix = lspace:5 rspace:5 # &Therefore;
-operator.\u2758.infix = direction:vertical stretchy symmetric lspace:5 rspace:5 # &VerticalSeparator;
-operator.\u002F\u002F.infix = lspace:5 rspace:5 # //
-operator.\u2237.infix = lspace:5 rspace:5 # &Colon; &Proportion;
-operator.\u0026.prefix = lspace:0 rspace:5 # &amp;
+operator.\u0021.postfix = lspace:2 rspace:0 # !
+operator.\u0021\u0021.postfix = lspace:2 rspace:0 # !!
+operator.\u0021\u003D.infix = lspace:5 rspace:5 # !=
 operator.\u0026.postfix = lspace:5 rspace:0 # &amp;
+operator.\u0026\u0026.infix = lspace:5 rspace:5 # &amp;&amp;
+operator.\u0027.postfix = lspace:2 rspace:0 # '
+operator.\u0028.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # (
+operator.\u0029.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # )
+operator.\u002A.infix = lspace:3 rspace:3 # *
+operator.\u002A\u002A.infix = lspace:2 rspace:2 # **
 operator.\u002A\u003D.infix = lspace:5 rspace:5 # *=
+operator.\u002B.infix = lspace:4 rspace:4 # +
+operator.\u002B.prefix = lspace:0 rspace:1 # +
+operator.\u002B\u002B.postfix = lspace:2 rspace:0 # ++
+operator.\u002B\u003D.infix = lspace:5 rspace:5 # +=
+operator.\u002C.infix = lspace:0 rspace:6 separator # ,
+operator.\u002D.infix = lspace:4 rspace:4 # -
+operator.\u002D.prefix = lspace:0 rspace:1 # -
+operator.\u002D\u002D.postfix = lspace:2 rspace:0 # --
 operator.\u002D\u003D.infix = lspace:5 rspace:5 # -=
-operator.\u002B\u003D.infix = lspace:5 rspace:5 # +=
-operator.\u002F\u003D.infix = lspace:5 rspace:5 # /=
 operator.\u002D\u003E.infix = lspace:5 rspace:5 # ->
-operator.\u003A.infix = lspace:5 rspace:5 # :
+operator.\u002E.infix = lspace:0 rspace:0 # .
 operator.\u002E\u002E.postfix = lspace:4 rspace:0 # ..
 operator.\u002E\u002E\u002E.postfix = lspace:4 rspace:0 # ...
-operator.\u220B.infix = lspace:5 rspace:5 # &SuchThat; &ReverseElement;
-operator.\u2AE4.infix = lspace:5 rspace:5 # &DoubleLeftTee;
-operator.\u22A8.infix = lspace:5 rspace:5 # &DoubleRightTee;
-operator.\u22A4.infix = lspace:5 rspace:5 # &DownTee;
-operator.\u22A3.infix = lspace:5 rspace:5 # &LeftTee;
-operator.\u22A2.infix = lspace:5 rspace:5 # &RightTee;
-operator.\u21D2.infix = direction:horizontal stretchy lspace:5 rspace:5 # &Implies; &DoubleRightArrow;
-operator.\u2970.infix = lspace:5 rspace:5 # &RoundImplies;
+operator.\u002F.infix = lspace:3 rspace:3 stretchy direction:vertical # /
+operator.\u002F\u002F.infix = lspace:5 rspace:5 # //
+operator.\u002F\u003D.infix = lspace:5 rspace:5 # /=
+operator.\u003A.infix = lspace:5 rspace:5 # :
+operator.\u003A\u003D.infix = lspace:5 rspace:5 # :=
+operator.\u003B.infix = lspace:0 rspace:5 separator # ;
+operator.\u003C.infix = lspace:5 rspace:5 # &lt;
+operator.\u003C\u003D.infix = lspace:5 rspace:5 # &lt;=
+operator.\u003C\u003E.infix = lspace:2 rspace:2 # &lt;>
+operator.\u003D.infix = lspace:5 rspace:5 # =
+operator.\u003D\u003D.infix = lspace:5 rspace:5 # ==
+operator.\u003E.infix = lspace:5 rspace:5 # >
+operator.\u003E\u003D.infix = lspace:5 rspace:5 # >=
+operator.\u003F.infix = lspace:2 rspace:2 # ?
+operator.\u0040.infix = lspace:2 rspace:2 # @
+operator.\u005B.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # [
+operator.\u005D.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # ]
+operator.\u005E.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &Hat; circumflex accent
+operator.\u005E.infix = lspace:2 rspace:2 direction:horizontal # ^
+operator.\u005F.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # _ low line
+operator.\u005F.infix = lspace:2 rspace:2 stretchy direction:horizontal # _
+operator.\u0060.postfix = lspace:0 rspace:0 accent # &DiacriticalGrave;
+operator.\u007B.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # {
+operator.\u007C.infix = lspace:0 rspace:0 stretchy symmetric direction:vertical # &VerticalLine; |
+operator.\u007C.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # |
+operator.\u007C.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # |
 operator.\u007C\u007C.infix = lspace:4 rspace:4 # ||
-operator.\u2A54.infix = direction:vertical lspace:4 rspace:4 # &Or;
-operator.\u0026\u0026.infix = lspace:5 rspace:5 # &amp;&amp;
-operator.\u2A53.infix = direction:vertical lspace:4 rspace:4 # &And;
-operator.\u0026.infix = lspace:5 rspace:5 # &amp;
-operator.\u0021.prefix = lspace:0 rspace:5 # !
-operator.\u2AEC.prefix = lspace:0 rspace:5 # &Not;
-operator.\u2203.prefix = lspace:0 rspace:5 # &Exists;
-operator.\u2200.prefix = lspace:0 rspace:5 # &ForAll;
-operator.\u2204.prefix = lspace:0 rspace:5 # &NotExists;
-operator.\u2208.infix = lspace:5 rspace:5 # &Element;
-operator.\u2209.infix = lspace:5 rspace:5 # &NotElement;
-operator.\u220C.infix = lspace:5 rspace:5 # &NotReverseElement;
-operator.\u228F\u0338.infix = lspace:5 rspace:5 # &NotSquareSubset;
-operator.\u22E2.infix = lspace:5 rspace:5 # &NotSquareSubsetEqual;
-operator.\u2290\u0338.infix = lspace:5 rspace:5 # &NotSquareSuperset;
-operator.\u22E3.infix = lspace:5 rspace:5 # &NotSquareSupersetEqual;
-operator.\u2284.infix = lspace:5 rspace:5 # &nsub;
-operator.\u228A.infix = lspace:5 rspace:5 # &subsetneq; &subne;
-operator.\u2282\u020D2.infix = lspace:5 rspace:5 # &NotSubset;
-operator.\u2288.infix = lspace:5 rspace:5 # &NotSubsetEqual;
-operator.\u2AC5\u0338.infix = lspace:5 rspace:5 # &nsubseteqq;
-operator.\u2285.infix = lspace:5 rspace:5 # &nsup;
-operator.\u2283\u020D2.infix = lspace:5 rspace:5 # &NotSuperset;
-operator.\u2289.infix = lspace:5 rspace:5 # &NotSupersetEqual;
-operator.\u2AC6\u0338.infix = lspace:5 rspace:5 # &nsubseteqq;
-operator.\u228F.infix = lspace:5 rspace:5 # &SquareSubset;
-operator.\u2291.infix = lspace:5 rspace:5 # &SquareSubsetEqual;
-operator.\u2290.infix = lspace:5 rspace:5 # &SquareSuperset;
-operator.\u2292.infix = lspace:5 rspace:5 # &SquareSupersetEqual;
-operator.\u22D0.infix = lspace:5 rspace:5 # &Subset;
-operator.\u2286.infix = lspace:5 rspace:5 # &SubsetEqual;
-operator.\u2AC5.infix = lspace:5 rspace:5 # &subseteqq;
-operator.\u2283.infix = lspace:5 rspace:5 # &Superset;
-operator.\u2287.infix = lspace:5 rspace:5 # &SupersetEqual;
-operator.\u2AC6.infix = lspace:5 rspace:5 # &supseteqq;
-operator.\u21D0.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DoubleLeftArrow;
-operator.\u21D4.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DoubleLeftRightArrow;
-operator.\u2950.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DownLeftRightVector;
-operator.\u295E.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DownLeftTeeVector;
-operator.\u21BD.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DownLeftVector;
-operator.\u2956.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DownLeftVectorBar;
-operator.\u295F.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DownRightTeeVector;
-operator.\u21C1.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DownRightVector;
-operator.\u2957.infix = direction:horizontal stretchy lspace:5 rspace:5 # &DownRightVectorBar;
-operator.\u2190.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftArrow;
-operator.\u21E4.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftArrowBar;
-operator.\u21C6.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftArrowRightArrow;
-operator.\u2194.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftRightArrow;
-operator.\u294E.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftRightVector;
-operator.\u21A4.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftTeeArrow;
-operator.\u295A.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftTeeVector;
-operator.\u21BC.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftVector;
-operator.\u2952.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LeftVectorBar;
-operator.\u2199.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LowerLeftArrow;
-operator.\u2198.infix = direction:horizontal stretchy lspace:5 rspace:5 # &LowerRightArrow;
-operator.\u2192.infix = direction:horizontal stretchy lspace:5 rspace:5 # &RightArrow;
-operator.\u21E5.infix = direction:horizontal stretchy lspace:5 rspace:5 # &RightArrowBar;
-operator.\u21C4.infix = direction:horizontal stretchy lspace:5 rspace:5 # &RightArrowLeftArrow;
-operator.\u21A6.infix = direction:horizontal stretchy lspace:5 rspace:5 # &RightTeeArrow;
-operator.\u295B.infix = direction:horizontal stretchy lspace:5 rspace:5 # &RightTeeVector;
-operator.\u21C0.infix = direction:horizontal stretchy lspace:5 rspace:5 # &RightVector;
-operator.\u2953.infix = direction:horizontal stretchy lspace:5 rspace:5 # &RightVectorBar;
-operator.\u2190\u200B.infix = lspace:5 rspace:5 # &ShortLeftArrow;
-operator.\u2192\u200B.infix = lspace:5 rspace:5 # &ShortRightArrow;
-operator.\u2196.infix = direction:vertical stretchy lspace:5 rspace:5 # &UpperLeftArrow;
-operator.\u2197.infix = direction:vertical stretchy lspace:5 rspace:5 # &UpperRightArrow;
-operator.\u003D.infix = lspace:5 rspace:5 # =
-operator.\u003C.infix = lspace:5 rspace:5 # &lt;
-operator.\u003E.infix = lspace:5 rspace:5 # >
-operator.\u0021\u003D.infix = lspace:5 rspace:5 # !=
-operator.\u003D\u003D.infix = lspace:5 rspace:5 # ==
-operator.\u003C\u003D.infix = lspace:5 rspace:5 # &lt;=
-operator.\u003E\u003D.infix = lspace:5 rspace:5 # >=
-operator.\u2261.infix = lspace:5 rspace:5 # &Congruent;
-operator.\u224D.infix = lspace:5 rspace:5 # &CupCap;
-operator.\u2250.infix = lspace:5 rspace:5 # &DotEqual;
-operator.\u2225.infix = direction:vertical stretchy lspace:5 rspace:5 # &DoubleVerticalBar;
-operator.\u2016.infix = direction:vertical stretchy lspace:5 rspace:5 # &Vert; &Verbar;
-operator.\u2A75.infix = lspace:5 rspace:5 # &Equal;
-operator.\u2242.infix = lspace:5 rspace:5 # &EqualTilde;
-operator.\u21CC.infix = direction:horizontal stretchy lspace:5 rspace:5 # &Equilibrium;
-operator.\u2265.infix = lspace:5 rspace:5 # &GreaterEqual;
-operator.\u22DB.infix = lspace:5 rspace:5 # &GreaterEqualLess;
-operator.\u2A8C.infix = lspace:5 rspace:5 # &gtreqqless;
-operator.\u2267.infix = lspace:5 rspace:5 # &GreaterFullEqual;
-operator.\u2AA2.infix = lspace:5 rspace:5 # &GreaterGreater;
-operator.\u2277.infix = lspace:5 rspace:5 # &GreaterLess;
-operator.\u2A7E.infix = lspace:5 rspace:5 # &GreaterSlantEqual;
-operator.\u2273.infix = lspace:5 rspace:5 # &GreaterTilde;
-operator.\u2A86.infix = lspace:5 rspace:5 # &gtrapprox;
-operator.\u224E.infix = lspace:5 rspace:5 # &HumpDownHump;
-operator.\u224F.infix = lspace:5 rspace:5 # &HumpEqual;
-operator.\u22B2.infix = lspace:5 rspace:5 # &LeftTriangle;
-operator.\u29CF.infix = lspace:5 rspace:5 # &LeftTriangleBar;
-operator.\u22B4.infix = lspace:5 rspace:5 # &LeftTriangleEqual;
-operator.\u2264.infix = lspace:5 rspace:5 # &le;
-operator.\u22DA.infix = lspace:5 rspace:5 # &LessEqualGreater;
-operator.\u2A8B.infix = lspace:5 rspace:5 # &lesseqqgtr;
-operator.\u2266.infix = lspace:5 rspace:5 # &LessFullEqual;
-operator.\u2276.infix = lspace:5 rspace:5 # &LessGreater;
-operator.\u2AA1.infix = lspace:5 rspace:5 # &LessLess;
-operator.\u2A7D.infix = lspace:5 rspace:5 # &LessSlantEqual;
-operator.\u2272.infix = lspace:5 rspace:5 # &LessTilde;
-operator.\u2A85.infix = lspace:5 rspace:5 # &lessapprox;
-operator.\u226B.infix = lspace:5 rspace:5 # &NestedGreaterGreater;
-operator.\u226A.infix = lspace:5 rspace:5 # &NestedLessLess;
-operator.\u2262.infix = lspace:5 rspace:5 # &NotCongruent;
-operator.\u226D.infix = lspace:5 rspace:5 # &NotCupCap;
-operator.\u2226.infix = lspace:5 rspace:5 # &NotDoubleVerticalBar;
-operator.\u2260.infix = lspace:5 rspace:5 # &NotEqual;
-operator.\u2242\u0338.infix = lspace:5 rspace:5 # &NotEqualTilde;
-operator.\u226F.infix = lspace:5 rspace:5 # &NotGreater;
-operator.\u003E\u20D2.infix = lspace:5 rspace:5 # &nvgt;
-operator.\u2271.infix = lspace:5 rspace:5 # &NotGreaterEqual;
-operator.\u2266\u0338.infix = lspace:5 rspace:5 # &NotGreaterFullEqual;
-operator.\u2A7D\u0338.infix = lspace:5 rspace:5 # &NotLessSlantEqual;
-operator.\u226B\u0338.infix = lspace:5 rspace:5 # &NotGreaterGreater;
-operator.\u2279.infix = lspace:5 rspace:5 # &NotGreaterLess;
-operator.\u2A7E\u0338.infix = lspace:5 rspace:5 # &NotGreaterSlantEqual;
-operator.\u2275.infix = lspace:5 rspace:5 # &NotGreaterTilde;
-operator.\u224E\u0338.infix = lspace:5 rspace:5 # &NotHumpDownHump;
-operator.\u224F\u0338.infix = lspace:5 rspace:5 # &NotHumpEqual;
-operator.\u22EA.infix = lspace:5 rspace:5 # &NotLeftTriangle;
-operator.\u29CF\u0338.infix = lspace:5 rspace:5 # &NotLeftTriangleBar;
-operator.\u22EC.infix = lspace:5 rspace:5 # &NotLeftTriangleEqual;
-operator.\u226E.infix = lspace:5 rspace:5 # &NotLess;
-operator.\u003C\u20D2.infix = lspace:5 rspace:5 # &nvlt;
-operator.\u2270.infix = lspace:5 rspace:5 # &NotLessEqual;
-# There is no &NotLessFullEqual;
-operator.\u2278.infix = lspace:5 rspace:5 # &NotLessGreater;
-operator.\u226A\u0338.infix = lspace:5 rspace:5 # &NotLessLess;
-operator.\u2274.infix = lspace:5 rspace:5 # &NotLessTilde;
-operator.\u2AA2\u0338.infix = lspace:5 rspace:5 # &NotNestedGreaterGreater;
-operator.\u2AA1\u0338.infix = lspace:5 rspace:5 # &NotNestedLessLess;
-operator.\u2280.infix = lspace:5 rspace:5 # &NotPrecedes;
-operator.\u2AAF\u0338.infix = lspace:5 rspace:5 # &NotPrecedesEqual;
-operator.\u22E0.infix = lspace:5 rspace:5 # &NotPrecedesSlantEqual;
-operator.\u22EB.infix = lspace:5 rspace:5 # &NotRightTriangle;
-operator.\u29D0\u0338.infix = lspace:5 rspace:5 # &NotRightTriangleBar;
-operator.\u22ED.infix = lspace:5 rspace:5 # &NotRightTriangleEqual;
-operator.\u2281.infix = lspace:5 rspace:5 # &NotSucceeds;
-operator.\u2AB0\u0338.infix = lspace:5 rspace:5 # &NotSucceedsEqual;
-operator.\u22E1.infix = lspace:5 rspace:5 # &NotSucceedsSlantEqual;
-operator.\u227F\u0338.infix = lspace:5 rspace:5 # &NotSucceedsTilde;
-operator.\u2241.infix = lspace:5 rspace:5 # &NotTilde;
-operator.\u2244.infix = lspace:5 rspace:5 # &NotTildeEqual;
-operator.\u2247.infix = lspace:5 rspace:5 # &NotTildeFullEqual;
-operator.\u2249.infix = lspace:5 rspace:5 # &NotTildeTilde;
-operator.\u2224.infix = lspace:5 rspace:5 # &NotVerticalBar;
-operator.\u227A.infix = lspace:5 rspace:5 # &Precedes;
-operator.\u2AAF.infix = lspace:5 rspace:5 # &PrecedesEqual;
-operator.\u227C.infix = lspace:5 rspace:5 # &PrecedesSlantEqual;
-operator.\u227E.infix = lspace:5 rspace:5 # &PrecedesTilde;
-operator.\u2AB3.infix = lspace:5 rspace:5 # &prE;
-operator.\u2AB7.infix = lspace:5 rspace:5 # &precapprox;
-operator.\u221D.infix = lspace:5 rspace:5 # &Proportional;
-operator.\u21CB.infix = direction:horizontal stretchy lspace:5 rspace:5 # &ReverseEquilibrium;
-operator.\u22B3.infix = lspace:5 rspace:5 # &RightTriangle;
-operator.\u29D0.infix = lspace:5 rspace:5 # &RightTriangleBar;
-operator.\u22B5.infix = lspace:5 rspace:5 # &RightTriangleEqual;
-operator.\u227B.infix = lspace:5 rspace:5 # &Succeeds;
-operator.\u2AB0.infix = lspace:5 rspace:5 # &SucceedsEqual;
-operator.\u227D.infix = lspace:5 rspace:5 # &SucceedsSlantEqual;
-operator.\u227F.infix = lspace:5 rspace:5 # &SucceedsTilde;
-operator.\u2AB4.infix = lspace:5 rspace:5 # &scE;
-operator.\u2AB8.infix = lspace:5 rspace:5 # &succapprox;
-operator.\u223C.infix = lspace:5 rspace:5 # &Tilde;
-operator.\u2243.infix = lspace:5 rspace:5 # &TildeEqual;
-operator.\u2245.infix = lspace:5 rspace:5 # &TildeFullEqual;
-operator.\u2248.infix = lspace:5 rspace:5 # &TildeTilde;
-operator.\u22A5.infix = lspace:5 rspace:5 # &UpTee;
-operator.\u2223.infix = direction:vertical stretchy lspace:5 rspace:5 # &VerticalBar;
-operator.\u2294.infix = direction:vertical lspace:4 rspace:4 # &SquareUnion;
-operator.\u228E.infix = direction:vertical lspace:4 rspace:4 # &UnionPlus;
-operator.\u002D.infix = lspace:4 rspace:4 # -
-operator.\u002B.infix = lspace:4 rspace:4 # +
-operator.\u2213.infix = lspace:4 rspace:4 # &MinusPlus;
+operator.\u007D.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # }
+operator.\u007E.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # ~ tilde
+operator.\u00A8.postfix = lspace:0 rspace:0 accent # &DoubleDot;
+operator.\u00AF.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &OverBar;
 operator.\u00B1.infix = lspace:4 rspace:4 # &PlusMinus;
-operator.\u2293.infix = direction:vertical lspace:4 rspace:4 # &SquareIntersection;
-operator.\u22C1.prefix = direction:vertical largeop symmetric movablelimits lspace:1 rspace:2 # &Vee;
-operator.\u2296.prefix = largeop movablelimits symmetric lspace:0 rspace:3 # &CircleMinus;
-operator.\u2295.prefix = largeop movablelimits symmetric lspace:0 rspace:3 # &CirclePlus;
-operator.\u2A01.prefix = largeop movablelimits symmetric lspace:0 rspace:3 # &bigoplus;
-operator.\u2211.prefix = direction:vertical largeop movablelimits symmetric lspace:1 rspace:2 # &Sum;
-operator.\u22C3.prefix = direction:vertical largeop movablelimits symmetric lspace:0 rspace:3 # &Union;
-operator.\u228E.prefix = direction:vertical largeop movablelimits symmetric lspace:1 rspace:2 # &UnionPlus;
-operator.\u2A04.prefix = direction:vertical largeop movablelimits symmetric lspace:1 rspace:2 # &biguplus;
-operator.\u2A06.prefix = direction:vertical largeop movablelimits symmetric lspace:1 rspace:2 # &bigsqcup;
-operator.\u006C\u0069\u006D.prefix = movablelimits lspace:0 rspace:3 # lim
-operator.\u006D\u0061\u0078.prefix = movablelimits lspace:0 rspace:3 # max
-operator.\u006D\u0069\u006E.prefix = movablelimits lspace:0 rspace:3 # min
-operator.\u2212.infix = lspace:4 rspace:4 # official Unicode minus sign
-operator.\u2212.prefix = lspace:0 rspace:1 # official Unicode minus sign
-operator.\u2296.infix = lspace:3 rspace:3 # &CircleMinus;
-operator.\u2295.infix = lspace:3 rspace:3 # &CirclePlus;
-operator.\u2232.prefix = direction:vertical largeop symmetric lspace:0 rspace:1 integral # &ClockwiseContourIntegral;
-operator.\u222E.prefix = direction:vertical largeop symmetric lspace:0 rspace:1 integral # &ContourIntegral;
-operator.\u2233.prefix = direction:vertical largeop symmetric lspace:0 rspace:1 integral # &CounterClockwiseContourIntegral;
-operator.\u222F.prefix = direction:vertical largeop symmetric lspace:0 rspace:1 integral # &DoubleContourIntegral;
-operator.\u222B.prefix = direction:vertical largeop symmetric lspace:0 rspace:1 integral # &Integral;
-operator.\u2A0B.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # summation with integral
-operator.\u222C.prefix = lspace:0 rspace:1 symmetric largeop integral direction:vertical # double integral
-operator.\u222D.prefix = lspace:0 rspace:1 symmetric largeop integral direction:vertical # triple integral
-operator.\u2230.prefix = lspace:0 rspace:1 symmetric largeop integral direction:vertical # volume integral
-operator.\u2231.prefix = lspace:0 rspace:1 symmetric largeop integral direction:vertical # clockwise integral
-operator.\u2A0C.prefix = lspace:0 rspace:1 symmetric largeop integral direction:vertical # quadruple integral operator
-operator.\u2A0D.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # finite part integral
-operator.\u2A0E.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral with double stroke
-operator.\u2A0F.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral average with slash
-operator.\u2A10.prefix = lspace:1 rspace:2 symmetric largeop movablelimits integral direction:vertical # circulation function
-operator.\u2A11.prefix = lspace:1 rspace:2 symmetric largeop movablelimits integral direction:vertical # anticlockwise integration
-operator.\u2A12.prefix = lspace:1 rspace:2 symmetric largeop movablelimits integral direction:vertical # line integration with rectangular path around pole
-operator.\u2A13.prefix = lspace:1 rspace:2 symmetric largeop movablelimits integral direction:vertical # line integration with semicircular path around pole
-operator.\u2A14.prefix = lspace:1 rspace:2 symmetric largeop movablelimits integral direction:vertical # line integration not including the pole
-operator.\u2A15.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral around a point operator
-operator.\u2A16.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # quaternion integral operator
-operator.\u2A17.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral with leftwards arrow with hook
-operator.\u2A18.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral with times sign
-operator.\u2A19.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral with intersection
-operator.\u2A1A.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral with union
-operator.\u2A1B.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral with overbar
-operator.\u2A1C.prefix = lspace:1 rspace:2 symmetric largeop integral direction:vertical # integral with underbar
-operator.\u22D3.infix = lspace:3 rspace:3 # &Cup;
-operator.\u22D2.infix = lspace:3 rspace:3 # &Cap;
-operator.\u2240.infix = lspace:3 rspace:3 # &VerticalTilde;
-operator.\u22C0.prefix = direction:vertical largeop movablelimits symmetric lspace:1 rspace:2 # &Wedge;
-operator.\u2297.prefix = largeop movablelimits symmetric lspace:0 rspace:3 # &CircleTimes;
-operator.\u2A02.prefix = largeop movablelimits symmetric lspace:0 rspace:3 # &bigotimes;
-operator.\u2210.prefix = direction:vertical largeop movablelimits symmetric lspace:1 rspace:2 # &Coproduct;
-operator.\u220F.prefix = direction:vertical largeop movablelimits symmetric lspace:1 rspace:2 # &Product;
-operator.\u22C2.prefix = direction:vertical largeop movablelimits symmetric lspace:1 rspace:2 # &Intersection;
-operator.\u2210.infix = lspace:3 rspace:3 direction:vertical # &Coproduct;
-operator.\u2606.infix = lspace:3 rspace:3 # &star;
-operator.\u22C6.infix = lspace:3 rspace:3 # &Star;
-operator.\u2299.prefix = largeop movablelimits symmetric lspace:0 rspace:3 # &CircleDot;
-operator.\u2A00.prefix = largeop movablelimits symmetric lspace:0 rspace:3 # &bigodot;
-operator.\u002A.infix = lspace:3 rspace:3 # *
+operator.\u00B1.prefix = lspace:0 rspace:1 # &PlusMinus;
+operator.\u00B4.postfix = lspace:0 rspace:0 accent # &DiacriticalAcute;
+operator.\u00B7.infix = lspace:3 rspace:3 # &CenterDot;
+operator.\u00B8.postfix = lspace:0 rspace:0 accent # &Cedilla;
+operator.\u02C6.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # modifier letter circumflex accent
+operator.\u02C7.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &Hacek; caron
+operator.\u02D8.postfix = lspace:0 rspace:0 accent # &Breve;
+operator.\u02D9.postfix = lspace:0 rspace:0 accent # &DiacriticalDot;
+operator.\u02DC.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &DiacriticalTilde; small tilde
+operator.\u02DD.postfix = lspace:0 rspace:0 accent # &DiacriticalDoubleAcute;
+operator.\u0311.postfix = lspace:0 rspace:0 accent # &DownBreve;
+operator.\u2016.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &Vert; &Verbar;
+operator.\u2016.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &Vert; &Verbar;
+operator.\u2018.prefix = lspace:0 rspace:0 fence # &OpenCurlyQuote;
+operator.\u2019.postfix = lspace:0 rspace:0 fence # &CloseCurlyQuote;
+operator.\u201C.prefix = lspace:0 rspace:0 fence # &OpenCurlyDoubleQuote;
+operator.\u201D.postfix = lspace:0 rspace:0 fence # &CloseCurlyDoubleQuote;
+operator.\u203E.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # overline
+operator.\u2061.infix = lspace:0 rspace:0 # &ApplyFunction;
 operator.\u2062.infix = lspace:0 rspace:0 # &InvisibleTimes;
-operator.\u00B7.infix = lspace:3 rspace:3 # &CenterDot;
-operator.\u2297.infix = lspace:3 rspace:3 # &CircleTimes;
-operator.\u22C1.infix = lspace:3 rspace:3 direction:vertical # &Vee;
-operator.\u22C0.infix = lspace:3 rspace:3 direction:vertical # &Wedge;
-operator.\u22C4.infix = lspace:3 rspace:3 # &Diamond;
-operator.\u2216.infix = direction:vertical stretchy lspace:3 rspace:3 # &Backslash;
-operator.\u002F.infix = direction:vertical stretchy lspace:3 rspace:3 # /
-operator.\u002D.prefix = lspace:0 rspace:1 # -
-operator.\u002B.prefix = lspace:0 rspace:1 # +
-operator.\u2213.prefix = lspace:0 rspace:1 # &MinusPlus;
-operator.\u00B1.prefix = lspace:0 rspace:1 # &PlusMinus;
-operator.\u002E.infix = lspace:0 rspace:0 # .
-operator.\u2A2F.infix = lspace:2 rspace:2 # &Cross;
-operator.\u002A\u002A.infix = lspace:2 rspace:2 # **
-operator.\u2299.infix = lspace:2 rspace:2 # &CircleDot;
-operator.\u2218.infix = lspace:2 rspace:2 # &SmallCircle;
-operator.\u25A1.prefix = lspace:0 rspace:2 # &Square;
-operator.\u2207.prefix = lspace:0 rspace:2 # &Del;
-operator.\u2202.prefix = lspace:0 rspace:2 # &PartialD;
+operator.\u2063.infix = lspace:0 rspace:0 separator # &InvisibleComma;
+operator.\u20DB.postfix = lspace:0 rspace:0 accent # &TripleDot;
 operator.\u2145.prefix = lspace:0 rspace:2 # &CapitalDifferentialD;
 operator.\u2146.prefix = lspace:0 rspace:2 # &DifferentialD;
-operator.\u221A.prefix = direction:vertical stretchy lspace:0 rspace:2 # &Sqrt;
-operator.\u21D3.infix = direction:vertical stretchy lspace:2 rspace:2 # &DoubleDownArrow;
-operator.\u27F8.infix = direction:horizontal stretchy lspace:2 rspace:2 # &DoubleLongLeftArrow;
-operator.\u27FA.infix = direction:horizontal stretchy lspace:2 rspace:2 # &DoubleLongLeftRightArrow;
-operator.\u27F9.infix = direction:horizontal stretchy lspace:2 rspace:2 # &DoubleLongRightArrow;
-operator.\u21D1.infix = direction:vertical stretchy lspace:2 rspace:2 # &DoubleUpArrow;
-operator.\u21D5.infix = direction:vertical stretchy lspace:2 rspace:2 # &DoubleUpDownArrow;
-operator.\u2193.infix = direction:vertical stretchy lspace:2 rspace:2 # &DownArrow;
-operator.\u2913.infix = direction:vertical stretchy lspace:2 rspace:2 # &DownArrowBar;
-operator.\u21F5.infix = direction:vertical stretchy lspace:2 rspace:2 # &DownArrowUpArrow;
-operator.\u21A7.infix = direction:vertical stretchy lspace:2 rspace:2 # &DownTeeArrow;
-operator.\u2961.infix = direction:vertical stretchy lspace:2 rspace:2 # &LeftDownTeeVector;
-operator.\u21C3.infix = direction:vertical stretchy lspace:2 rspace:2 # &LeftDownVector;
-operator.\u2959.infix = direction:vertical stretchy lspace:2 rspace:2 # &LeftDownVectorBar;
-operator.\u2951.infix = direction:vertical stretchy lspace:2 rspace:2 # &LeftUpDownVector;
-operator.\u2960.infix = direction:vertical stretchy lspace:2 rspace:2 # &LeftUpTeeVector;
-operator.\u21BF.infix = direction:vertical stretchy lspace:2 rspace:2 # &LeftUpVector;
-operator.\u2958.infix = direction:vertical stretchy lspace:2 rspace:2 # &LeftUpVectorBar;
-operator.\u27F5.infix = direction:horizontal stretchy lspace:2 rspace:2 # &LongLeftArrow;
-operator.\u27F7.infix = direction:horizontal stretchy lspace:2 rspace:2 # &LongLeftRightArrow;
-operator.\u27F6.infix = direction:horizontal stretchy lspace:2 rspace:2 # &LongRightArrow;
-operator.\u296F.infix = direction:vertical stretchy lspace:2 rspace:2 # &ReverseUpEquilibrium;
-operator.\u295D.infix = direction:vertical stretchy lspace:2 rspace:2 # &RightDownTeeVector;
-operator.\u21C2.infix = direction:vertical stretchy lspace:2 rspace:2 # &RightDownVector;
-operator.\u2955.infix = direction:vertical stretchy lspace:2 rspace:2 # &RightDownVectorBar;
-operator.\u294F.infix = direction:vertical stretchy lspace:2 rspace:2 # &RightUpDownVector;
-operator.\u295C.infix = direction:vertical stretchy lspace:2 rspace:2 # &RightUpTeeVector;
-operator.\u21BE.infix = direction:vertical stretchy lspace:2 rspace:2 # &RightUpVector;
-operator.\u2954.infix = direction:vertical stretchy lspace:2 rspace:2 # &RightUpVectorBar;
-operator.\u2193\u200B.infix = lspace:2 rspace:2 # &ShortDownArrow;
-operator.\u2191\u200B.infix = lspace:2 rspace:2 # &ShortUpArrow;
-operator.\u2191.infix = direction:vertical stretchy lspace:2 rspace:2 # &UpArrow;
-operator.\u2912.infix = direction:vertical stretchy lspace:2 rspace:2 # &UpArrowBar;
-operator.\u21C5.infix = direction:vertical stretchy lspace:2 rspace:2 # &UpArrowDownArrow;
-operator.\u2195.infix = direction:vertical stretchy lspace:2 rspace:2 # &UpDownArrow;
-operator.\u296E.infix = direction:vertical stretchy lspace:2 rspace:2 # &UpEquilibrium;
-operator.\u21A5.infix = direction:vertical stretchy lspace:2 rspace:2 # &UpTeeArrow;
-operator.\u005E.infix = lspace:2 rspace:2 direction:horizontal # ^
-operator.\u003C\u003E.infix = lspace:2 rspace:2 # &lt;>
-operator.\u0027.postfix = lspace:2 rspace:0 # '
-operator.\u0021.postfix = lspace:2 rspace:0 # !
-operator.\u0021\u0021.postfix = lspace:2 rspace:0 # !!
-operator.\u007E.infix = direction:horizontal stretchy lspace:2 rspace:2 # ~
-operator.\u0040.infix = lspace:2 rspace:2 # @
-operator.\u002D\u002D.postfix = lspace:2 rspace:0 # --
-operator.\u002D\u002D.prefix = lspace:0 rspace:2 # --
-operator.\u002B\u002B.postfix = lspace:2 rspace:0 # ++
-operator.\u002B\u002B.prefix = lspace:0 rspace:2 # ++
-operator.\u2061.infix = lspace:0 rspace:0 # &ApplyFunction;
-operator.\u003F.infix = lspace:2 rspace:2 # ?
-operator.\u005F.infix = direction:horizontal stretchy lspace:2 rspace:2 # _
-operator.\u02D8.postfix = accent lspace:0 rspace:0 # &Breve;
-operator.\u00B8.postfix = accent lspace:0 rspace:0 # &Cedilla;
-operator.\u0060.postfix = accent lspace:0 rspace:0 # &DiacriticalGrave;
-operator.\u02D9.postfix = accent lspace:0 rspace:0 # &DiacriticalDot;
-operator.\u02DD.postfix = accent lspace:0 rspace:0 # &DiacriticalDoubleAcute;
-operator.\u00B4.postfix = accent lspace:0 rspace:0 # &DiacriticalAcute;
-operator.\u007E.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # ~ tilde
-operator.\u02DC.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &DiacriticalTilde; small tilde
-operator.\u00A8.postfix = accent lspace:0 rspace:0 # &DoubleDot;
-operator.\u0311.postfix = accent lspace:0 rspace:0 # &DownBreve;
-operator.\u02C7.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &Hacek; caron
-operator.\u005E.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &Hat; circumflex accent
-operator.\u02C6.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # modifier letter circumflex accent
-operator.\u00AF.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &OverBar;
-operator.\u203E.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # overline
-operator.\u23DE.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &OverBrace; (Unicode)
-operator.\uFE37.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &OverBrace; (MathML 2.0)
-operator.\u23B4.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &OverBracket;
-operator.\u23DC.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &OverParenthesis; (Unicode)
-operator.\uFE35.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &OverParenthesis; (MathML 2.0)
-operator.\u20DB.postfix = accent lspace:0 rspace:0 # &TripleDot;
-operator.\u005F.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # _ low line
-operator.\u0332.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &UnderBar;
-operator.\u23DF.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &UnderBrace; (Unicode)
-operator.\uFE38.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &UnderBrace; (MathML 2.0)
-operator.\u23B5.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &UnderBracket;
-operator.\u23DD.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &UnderParenthesis; (Unicode)
-operator.\uFE36.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &UnderParenthesis; (MathML 2.0)
-operator.\u2225.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &DoubleVerticalBar;
-operator.\u2225.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &DoubleVerticalBar;
-operator.\u2016.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &Vert; &Verbar;
-operator.\u2016.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &Vert; &Verbar;
-operator.\u2223.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &VerticalBar;
-operator.\u2223.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &VerticalBar;
-operator.\u007C.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # |
-operator.\u007C.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # |
-operator.\u20D7.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &#x20D7;
-operator.\u20D6.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &#x20D6;
-operator.\u20E1.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &#x20E1;
-operator.\u20D1.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &#x20D1;
-operator.\u20D0.postfix = direction:horizontal stretchy accent lspace:0 rspace:0 # &#x20D0;
-operator.\u21A9.infix = direction:horizontal stretchy lspace:5 rspace:5 # &hookleftarrow; &larrhk;
-operator.\u21AA.infix = direction:horizontal stretchy lspace:5 rspace:5 # &hookrightarrow; &rarrhk;
-operator.\u23B0.prefix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &lmoustache; &lmoust;
-operator.\u23B1.postfix = direction:vertical stretchy fence symmetric lspace:0 rspace:0 # &rmoustache; &rmoust;
-operator.\u22C5.infix = lspace:3 rspace:3 # &cdot;
-operator.\u2282.infix = lspace:5 rspace:5 # &subset;
-operator.\u2227.infix = lspace:4 rspace:4 # &wedge;
-operator.\u2228.infix = lspace:4 rspace:4 # &vee;
-operator.\u2229.infix = lspace:4 rspace:4 # &cap;
-operator.\u222A.infix = lspace:4 rspace:4 # &cup;
+operator.\u2190.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftArrow;
+operator.\u2191.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpArrow;
+operator.\u2192.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightArrow;
+operator.\u2193.infix = lspace:2 rspace:2 stretchy direction:vertical # &DownArrow;
+operator.\u2194.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftRightArrow;
+operator.\u2195.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpDownArrow;
+operator.\u2196.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpperLeftArrow;
+operator.\u2197.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpperRightArrow;
+operator.\u2198.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LowerRightArrow;
+operator.\u2199.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LowerLeftArrow;
 operator.\u219C.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards wave arrow
 operator.\u219D.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards wave arrow
 operator.\u219E.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards two headed arrow
 operator.\u219F.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards two headed arrow
 operator.\u21A0.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards two headed arrow
 operator.\u21A1.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards two headed arrow
 operator.\u21A2.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow with tail
 operator.\u21A3.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards arrow with tail
+operator.\u21A4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftTeeArrow;
+operator.\u21A5.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpTeeArrow;
+operator.\u21A6.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightTeeArrow;
+operator.\u21A7.infix = lspace:2 rspace:2 stretchy direction:vertical # &DownTeeArrow;
 operator.\u21A8.infix = lspace:5 rspace:5 stretchy direction:vertical # up down arrow with base
+operator.\u21A9.infix = lspace:5 rspace:5 stretchy direction:horizontal # &hookleftarrow; &larrhk;
+operator.\u21AA.infix = lspace:5 rspace:5 stretchy direction:horizontal # &hookrightarrow; &rarrhk;
 operator.\u21AB.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow with loop
 operator.\u21AC.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards arrow with loop
 operator.\u21AD.infix = lspace:5 rspace:5 stretchy direction:horizontal # left right wave arrow
 operator.\u21AF.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards zigzag arrow
 operator.\u21B0.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards arrow with tip leftwards
 operator.\u21B1.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards arrow with tip rightwards
 operator.\u21B2.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards arrow with tip leftwards
 operator.\u21B3.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards arrow with tip rightwards
 operator.\u21B4.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards arrow with corner downwards
 operator.\u21B5.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards arrow with corner leftwards
 operator.\u21B9.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow to bar over rightwards arrow to bar
+operator.\u21BC.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftVector;
+operator.\u21BD.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftVector;
+operator.\u21BE.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightUpVector;
+operator.\u21BF.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftUpVector;
+operator.\u21C0.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightVector;
+operator.\u21C1.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightVector;
+operator.\u21C2.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightDownVector;
+operator.\u21C3.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftDownVector;
+operator.\u21C4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightArrowLeftArrow;
+operator.\u21C5.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpArrowDownArrow;
+operator.\u21C6.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftArrowRightArrow;
 operator.\u21C7.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards paired arrows
 operator.\u21C8.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards paired arrows
 operator.\u21C9.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards paired arrows
 operator.\u21CA.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards paired arrows
+operator.\u21CB.infix = lspace:5 rspace:5 stretchy direction:horizontal # &ReverseEquilibrium;
+operator.\u21CC.infix = lspace:5 rspace:5 stretchy direction:horizontal # &Equilibrium;
+operator.\u21D0.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DoubleLeftArrow;
+operator.\u21D1.infix = lspace:2 rspace:2 stretchy direction:vertical # &DoubleUpArrow;
+operator.\u21D2.infix = lspace:5 rspace:5 stretchy direction:horizontal # &Implies; &DoubleRightArrow;
+operator.\u21D3.infix = lspace:2 rspace:2 stretchy direction:vertical # &DoubleDownArrow;
+operator.\u21D4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DoubleLeftRightArrow;
+operator.\u21D5.infix = lspace:2 rspace:2 stretchy direction:vertical # &DoubleUpDownArrow;
 operator.\u21DA.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards triple arrow
 operator.\u21DB.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards triple arrow
 operator.\u21DC.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards squiggle arrow
 operator.\u21DD.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards squiggle arrow
 operator.\u21E0.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards dashed arrow
 operator.\u21E1.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards dashed arrow
 operator.\u21E2.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards dashed arrow
 operator.\u21E3.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards dashed arrow
+operator.\u21E4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftArrowBar;
+operator.\u21E5.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightArrowBar;
 operator.\u21E6.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards white arrow
 operator.\u21E7.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow
 operator.\u21E8.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards white arrow
 operator.\u21E9.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards white arrow
 operator.\u21EA.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow from bar
 operator.\u21EB.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow on pedestal
 operator.\u21EC.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow on pedestal with horizontal bar
 operator.\u21ED.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow on pedestal with vertical bar
 operator.\u21EE.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white double arrow
 operator.\u21EF.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white double arrow on pedestal
 operator.\u21F0.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards white arrow from wall
 operator.\u21F3.infix = lspace:5 rspace:5 stretchy direction:vertical # up down white arrow
+operator.\u21F5.infix = lspace:2 rspace:2 stretchy direction:vertical # &DownArrowUpArrow;
 operator.\u21F6.infix = lspace:5 rspace:5 stretchy direction:horizontal # three rightwards arrows
 operator.\u21FD.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards open-headed arrow
 operator.\u21FE.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards open-headed arrow
 operator.\u21FF.infix = lspace:5 rspace:5 stretchy direction:horizontal # left right open-headed arrow
+operator.\u2200.prefix = lspace:0 rspace:5 # &ForAll;
+operator.\u2202.prefix = lspace:0 rspace:2 # &PartialD;
+operator.\u2203.prefix = lspace:0 rspace:5 # &Exists;
+operator.\u2204.prefix = lspace:0 rspace:5 # &NotExists;
+operator.\u2207.prefix = lspace:0 rspace:2 # &Del;
+operator.\u2208.infix = lspace:5 rspace:5 # &Element;
+operator.\u2209.infix = lspace:5 rspace:5 # &NotElement;
+operator.\u220B.infix = lspace:5 rspace:5 # &SuchThat; &ReverseElement;
+operator.\u220C.infix = lspace:5 rspace:5 # &NotReverseElement;
+operator.\u220F.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Product;
+operator.\u2210.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Coproduct;
+operator.\u2211.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Sum;
+operator.\u2212.infix = lspace:4 rspace:4 # official Unicode minus sign
+operator.\u2212.prefix = lspace:0 rspace:1 # official Unicode minus sign
+operator.\u2213.infix = lspace:4 rspace:4 # &MinusPlus;
+operator.\u2213.prefix = lspace:0 rspace:1 # &MinusPlus;
+operator.\u2216.infix = lspace:3 rspace:3 stretchy direction:vertical # &Backslash;
+operator.\u2218.infix = lspace:2 rspace:2 # &SmallCircle;
+operator.\u221A.prefix = lspace:0 rspace:2 stretchy direction:vertical # &Sqrt;
+operator.\u221D.infix = lspace:5 rspace:5 # &Proportional;
+operator.\u2223.infix = lspace:5 rspace:5 stretchy direction:vertical # &VerticalBar;
+operator.\u2224.infix = lspace:5 rspace:5 # &NotVerticalBar;
+operator.\u2225.infix = lspace:5 rspace:5 stretchy direction:vertical # &DoubleVerticalBar;
+operator.\u2226.infix = lspace:5 rspace:5 # &NotDoubleVerticalBar;
+operator.\u2227.infix = lspace:4 rspace:4 # &wedge;
+operator.\u2228.infix = lspace:4 rspace:4 # &vee;
+operator.\u2229.infix = lspace:4 rspace:4 # &cap;
+operator.\u222A.infix = lspace:4 rspace:4 # &cup;
+operator.\u222B.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # &Integral;
+operator.\u222C.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # double integral
+operator.\u222D.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # triple integral
+operator.\u222E.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # &ContourIntegral;
+operator.\u222F.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # &DoubleContourIntegral;
+operator.\u2230.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # volume integral
+operator.\u2231.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # clockwise integral
+operator.\u2232.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # &ClockwiseContourIntegral;
+operator.\u2233.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # &CounterClockwiseContourIntegral;
+operator.\u2234.infix = lspace:5 rspace:5 # &Therefore;
+operator.\u2235.infix = lspace:5 rspace:5 # &Because;
+operator.\u2237.infix = lspace:5 rspace:5 # &Colon; &Proportion;
+operator.\u223C.infix = lspace:5 rspace:5 # &Tilde;
+operator.\u2240.infix = lspace:3 rspace:3 # &VerticalTilde;
+operator.\u2241.infix = lspace:5 rspace:5 # &NotTilde;
+operator.\u2242.infix = lspace:5 rspace:5 # &EqualTilde;
+operator.\u2242\u0338.infix = lspace:5 rspace:5 # &NotEqualTilde;
+operator.\u2243.infix = lspace:5 rspace:5 # &TildeEqual;
+operator.\u2244.infix = lspace:5 rspace:5 # &NotTildeEqual;
+operator.\u2245.infix = lspace:5 rspace:5 # &TildeFullEqual;
+operator.\u2247.infix = lspace:5 rspace:5 # &NotTildeFullEqual;
+operator.\u2248.infix = lspace:5 rspace:5 # &TildeTilde;
+operator.\u2249.infix = lspace:5 rspace:5 # &NotTildeTilde;
+operator.\u224D.infix = lspace:5 rspace:5 # &CupCap;
+operator.\u224E.infix = lspace:5 rspace:5 # &HumpDownHump;
+operator.\u224E\u0338.infix = lspace:5 rspace:5 # &NotHumpDownHump;
+operator.\u224F.infix = lspace:5 rspace:5 # &HumpEqual;
+operator.\u224F\u0338.infix = lspace:5 rspace:5 # &NotHumpEqual;
+operator.\u2250.infix = lspace:5 rspace:5 # &DotEqual;
+operator.\u2254.infix = lspace:5 rspace:5 # &Assign;
+operator.\u2260.infix = lspace:5 rspace:5 # &NotEqual;
+operator.\u2261.infix = lspace:5 rspace:5 # &Congruent;
+operator.\u2262.infix = lspace:5 rspace:5 # &NotCongruent;
+operator.\u2264.infix = lspace:5 rspace:5 # &le;
+operator.\u2265.infix = lspace:5 rspace:5 # &GreaterEqual;
+operator.\u2266.infix = lspace:5 rspace:5 # &LessFullEqual;
+operator.\u2266\u0338.infix = lspace:5 rspace:5 # &NotGreaterFullEqual;
+operator.\u2267.infix = lspace:5 rspace:5 # &GreaterFullEqual;
+operator.\u226A.infix = lspace:5 rspace:5 # &NestedLessLess;
+operator.\u226A\u0338.infix = lspace:5 rspace:5 # &NotLessLess;
+operator.\u226B.infix = lspace:5 rspace:5 # &NestedGreaterGreater;
+operator.\u226B\u0338.infix = lspace:5 rspace:5 # &NotGreaterGreater;
+operator.\u226D.infix = lspace:5 rspace:5 # &NotCupCap;
+operator.\u226E.infix = lspace:5 rspace:5 # &NotLess;
+operator.\u226F.infix = lspace:5 rspace:5 # &NotGreater;
+operator.\u2270.infix = lspace:5 rspace:5 # &NotLessEqual;
+operator.\u2271.infix = lspace:5 rspace:5 # &NotGreaterEqual;
+operator.\u2272.infix = lspace:5 rspace:5 # &LessTilde;
+operator.\u2273.infix = lspace:5 rspace:5 # &GreaterTilde;
+operator.\u2274.infix = lspace:5 rspace:5 # &NotLessTilde;
+operator.\u2275.infix = lspace:5 rspace:5 # &NotGreaterTilde;
+operator.\u2276.infix = lspace:5 rspace:5 # &LessGreater;
+operator.\u2277.infix = lspace:5 rspace:5 # &GreaterLess;
+operator.\u2278.infix = lspace:5 rspace:5 # &NotLessGreater;
+operator.\u2279.infix = lspace:5 rspace:5 # &NotGreaterLess;
+operator.\u227A.infix = lspace:5 rspace:5 # &Precedes;
+operator.\u227B.infix = lspace:5 rspace:5 # &Succeeds;
+operator.\u227C.infix = lspace:5 rspace:5 # &PrecedesSlantEqual;
+operator.\u227D.infix = lspace:5 rspace:5 # &SucceedsSlantEqual;
+operator.\u227E.infix = lspace:5 rspace:5 # &PrecedesTilde;
+operator.\u227F.infix = lspace:5 rspace:5 # &SucceedsTilde;
+operator.\u227F\u0338.infix = lspace:5 rspace:5 # &NotSucceedsTilde;
+operator.\u2280.infix = lspace:5 rspace:5 # &NotPrecedes;
+operator.\u2281.infix = lspace:5 rspace:5 # &NotSucceeds;
+operator.\u2282.infix = lspace:5 rspace:5 # &subset;
+operator.\u2283.infix = lspace:5 rspace:5 # &Superset;
+operator.\u2284.infix = lspace:5 rspace:5 # &nsub;
+operator.\u2285.infix = lspace:5 rspace:5 # &nsup;
+operator.\u2286.infix = lspace:5 rspace:5 # &SubsetEqual;
+operator.\u2287.infix = lspace:5 rspace:5 # &SupersetEqual;
+operator.\u2288.infix = lspace:5 rspace:5 # &NotSubsetEqual;
+operator.\u2289.infix = lspace:5 rspace:5 # &NotSupersetEqual;
+operator.\u228A.infix = lspace:5 rspace:5 # &subsetneq; &subne;
+operator.\u228E.infix = lspace:4 rspace:4 direction:vertical # &UnionPlus;
+operator.\u228F.infix = lspace:5 rspace:5 # &SquareSubset;
+operator.\u228F\u0338.infix = lspace:5 rspace:5 # &NotSquareSubset;
+operator.\u2290.infix = lspace:5 rspace:5 # &SquareSuperset;
+operator.\u2290\u0338.infix = lspace:5 rspace:5 # &NotSquareSuperset;
+operator.\u2291.infix = lspace:5 rspace:5 # &SquareSubsetEqual;
+operator.\u2292.infix = lspace:5 rspace:5 # &SquareSupersetEqual;
+operator.\u2293.infix = lspace:4 rspace:4 direction:vertical # &SquareIntersection;
+operator.\u2294.infix = lspace:4 rspace:4 direction:vertical # &SquareUnion;
+operator.\u2295.infix = lspace:3 rspace:3 # &CirclePlus;
+operator.\u2296.infix = lspace:3 rspace:3 # &CircleMinus;
+operator.\u2297.infix = lspace:3 rspace:3 # &CircleTimes;
+operator.\u2299.infix = lspace:2 rspace:2 # &CircleDot;
+operator.\u22A2.infix = lspace:5 rspace:5 # &RightTee;
+operator.\u22A3.infix = lspace:5 rspace:5 # &LeftTee;
+operator.\u22A4.infix = lspace:5 rspace:5 # &DownTee;
+operator.\u22A5.infix = lspace:5 rspace:5 # &UpTee;
+operator.\u22A8.infix = lspace:5 rspace:5 # &DoubleRightTee;
+operator.\u22B2.infix = lspace:5 rspace:5 # &LeftTriangle;
+operator.\u22B3.infix = lspace:5 rspace:5 # &RightTriangle;
+operator.\u22B4.infix = lspace:5 rspace:5 # &LeftTriangleEqual;
+operator.\u22B5.infix = lspace:5 rspace:5 # &RightTriangleEqual;
+operator.\u22C0.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Wedge;
+operator.\u22C1.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Vee;
+operator.\u22C2.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &Intersection;
+operator.\u22C3.prefix = lspace:0 rspace:3 largeop movablelimits symmetric direction:vertical # &Union;
+operator.\u22C4.infix = lspace:3 rspace:3 # &Diamond;
+operator.\u22C5.infix = lspace:3 rspace:3 # &cdot;
+operator.\u22C6.infix = lspace:3 rspace:3 # &Star;
+operator.\u22D0.infix = lspace:5 rspace:5 # &Subset;
+operator.\u22D2.infix = lspace:3 rspace:3 # &Cap;
+operator.\u22D3.infix = lspace:3 rspace:3 # &Cup;
+operator.\u22DA.infix = lspace:5 rspace:5 # &LessEqualGreater;
+operator.\u22DB.infix = lspace:5 rspace:5 # &GreaterEqualLess;
+operator.\u22E0.infix = lspace:5 rspace:5 # &NotPrecedesSlantEqual;
+operator.\u22E1.infix = lspace:5 rspace:5 # &NotSucceedsSlantEqual;
+operator.\u22E2.infix = lspace:5 rspace:5 # &NotSquareSubsetEqual;
+operator.\u22E3.infix = lspace:5 rspace:5 # &NotSquareSupersetEqual;
+operator.\u22EA.infix = lspace:5 rspace:5 # &NotLeftTriangle;
+operator.\u22EB.infix = lspace:5 rspace:5 # &NotRightTriangle;
+operator.\u22EC.infix = lspace:5 rspace:5 # &NotLeftTriangleEqual;
+operator.\u22ED.infix = lspace:5 rspace:5 # &NotRightTriangleEqual;
+operator.\u2308.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &LeftCeiling;
+operator.\u2309.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &RightCeiling;
+operator.\u230A.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &LeftFloor;
+operator.\u230B.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &RightFloor;
+operator.\u23B4.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &OverBracket;
+operator.\u23B5.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &UnderBracket;
+operator.\u23DC.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &OverParenthesis; (Unicode)
+operator.\u23DD.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &UnderParenthesis; (Unicode)
+operator.\u23DE.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &OverBrace; (Unicode)
+operator.\u23DF.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &UnderBrace; (Unicode)
+operator.\u2758.infix = lspace:5 rspace:5 stretchy symmetric direction:vertical # &VerticalSeparator;
+operator.\u27E6.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &LeftDoubleBracket;
+operator.\u27E7.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &RightDoubleBracket;
+operator.\u27E8.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &LeftAngleBracket;
+operator.\u27E9.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &RightAngleBracket;
+operator.\u27F5.infix = lspace:2 rspace:2 stretchy direction:horizontal # &LongLeftArrow;
+operator.\u27F6.infix = lspace:2 rspace:2 stretchy direction:horizontal # &LongRightArrow;
+operator.\u27F7.infix = lspace:2 rspace:2 stretchy direction:horizontal # &LongLeftRightArrow;
+operator.\u27F8.infix = lspace:2 rspace:2 stretchy direction:horizontal # &DoubleLongLeftArrow;
+operator.\u27F9.infix = lspace:2 rspace:2 stretchy direction:horizontal # &DoubleLongRightArrow;
+operator.\u27FA.infix = lspace:2 rspace:2 stretchy direction:horizontal # &DoubleLongLeftRightArrow;
 operator.\u290A.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards triple arrow
 operator.\u290B.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards triple arrow
 operator.\u290C.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards double dash arrow
 operator.\u290D.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards double dash arrow
 operator.\u290E.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards triple dash arrow
 operator.\u290F.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards triple dash arrow
 operator.\u2910.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards two-headed triple dash arrow
-operator.\u2991.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left angle bracket with dot
-operator.\u2992.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right angle bracket with dot
-operator.\u2993.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left arc less-than bracket
-operator.\u2994.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right arc greater-than bracket
-operator.\u2995.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # double left arc greater-than bracket
-operator.\u2996.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # double right arc less-than bracket
-operator.\u2983.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left white curly bracket
-operator.\u2984.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right white curly bracket
-operator.\u2985.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left white parenthesis
-operator.\u2986.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right white parenthesis
-operator.\u2987.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # z notation left image bracket
-operator.\u2988.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # z notation right image bracket
-operator.\u2989.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # z notation left binding bracket
-operator.\u298A.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # z notation right binding bracket
-operator.\u298B.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left square bracket with underbar
-operator.\u298C.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right square bracket with underbar
-operator.\u298D.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left square bracket with tick in top corner
-operator.\u298E.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right square bracket with tick in bottom corner
-operator.\u298F.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left square bracket with tick in bottom corner
-operator.\u2990.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right square bracket with tick in top corner
-operator.\u2997.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left black tortoise shell bracket
-operator.\u2998.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right black tortoise shell bracket
-operator.\u29FC.prefix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # left-pointing curved angle bracket
-operator.\u29FD.postfix = lspace:0 rspace:0 fence stretchy symmetric direction:vertical # right-pointing curved angle bracket
+operator.\u2912.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpArrowBar;
+operator.\u2913.infix = lspace:2 rspace:2 stretchy direction:vertical # &DownArrowBar;
+operator.\u294E.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftRightVector;
+operator.\u294F.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightUpDownVector;
+operator.\u2950.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftRightVector;
+operator.\u2951.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftUpDownVector;
+operator.\u2952.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftVectorBar;
+operator.\u2953.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightVectorBar;
+operator.\u2954.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightUpVectorBar;
+operator.\u2955.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightDownVectorBar;
+operator.\u2956.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftVectorBar;
+operator.\u2957.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightVectorBar;
+operator.\u2958.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftUpVectorBar;
+operator.\u2959.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftDownVectorBar;
+operator.\u295A.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftTeeVector;
+operator.\u295B.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightTeeVector;
+operator.\u295C.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightUpTeeVector;
+operator.\u295D.infix = lspace:2 rspace:2 stretchy direction:vertical # &RightDownTeeVector;
+operator.\u295E.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftTeeVector;
+operator.\u295F.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightTeeVector;
+operator.\u2960.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftUpTeeVector;
+operator.\u2961.infix = lspace:2 rspace:2 stretchy direction:vertical # &LeftDownTeeVector;
+operator.\u296E.infix = lspace:2 rspace:2 stretchy direction:vertical # &UpEquilibrium;
+operator.\u296F.infix = lspace:2 rspace:2 stretchy direction:vertical # &ReverseUpEquilibrium;
+operator.\u2970.infix = lspace:5 rspace:5 # &RoundImplies;
+operator.\u2983.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left white curly bracket
+operator.\u2984.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right white curly bracket
+operator.\u2985.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left white parenthesis
+operator.\u2986.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right white parenthesis
+operator.\u2987.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # z notation left image bracket
+operator.\u2988.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # z notation right image bracket
+operator.\u2989.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # z notation left binding bracket
+operator.\u298A.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # z notation right binding bracket
+operator.\u298B.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left square bracket with underbar
+operator.\u298C.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right square bracket with underbar
+operator.\u298D.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left square bracket with tick in top corner
+operator.\u298E.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right square bracket with tick in bottom corner
+operator.\u298F.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left square bracket with tick in bottom corner
+operator.\u2990.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right square bracket with tick in top corner
+operator.\u2991.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left angle bracket with dot
+operator.\u2992.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right angle bracket with dot
+operator.\u2993.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left arc less-than bracket
+operator.\u2994.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right arc greater-than bracket
+operator.\u2995.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # double left arc greater-than bracket
+operator.\u2996.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # double right arc less-than bracket
+operator.\u2997.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left black tortoise shell bracket
+operator.\u2998.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right black tortoise shell bracket
+operator.\u29CF.infix = lspace:5 rspace:5 # &LeftTriangleBar;
+operator.\u29CF\u0338.infix = lspace:5 rspace:5 # &NotLeftTriangleBar;
+operator.\u29D0.infix = lspace:5 rspace:5 # &RightTriangleBar;
+operator.\u29D0\u0338.infix = lspace:5 rspace:5 # &NotRightTriangleBar;
+operator.\u29FC.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # left-pointing curved angle bracket
+operator.\u29FD.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # right-pointing curved angle bracket
+operator.\u2A00.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &bigodot;
+operator.\u2A01.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &bigoplus;
+operator.\u2A02.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &bigotimes;
+operator.\u2A04.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &biguplus;
+operator.\u2A06.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &bigsqcup;
+operator.\u2A0B.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # summation with integral
+operator.\u2A0C.prefix = lspace:0 rspace:1 largeop symmetric direction:vertical integral # quadruple integral operator
+operator.\u2A0D.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # finite part integral
+operator.\u2A0E.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral with double stroke
+operator.\u2A0F.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral average with slash
+operator.\u2A10.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical integral # circulation function
+operator.\u2A11.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical integral # anticlockwise integration
+operator.\u2A12.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical integral # line integration with rectangular path around pole
+operator.\u2A13.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical integral # line integration with semicircular path around pole
+operator.\u2A14.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical integral # line integration not including the pole
+operator.\u2A15.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral around a point operator
+operator.\u2A16.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # quaternion integral operator
+operator.\u2A17.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral with leftwards arrow with hook
+operator.\u2A18.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral with times sign
+operator.\u2A19.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral with intersection
+operator.\u2A1A.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral with union
+operator.\u2A1B.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral with overbar
+operator.\u2A1C.prefix = lspace:1 rspace:2 largeop symmetric direction:vertical integral # integral with underbar
+operator.\u2A2F.infix = lspace:2 rspace:2 # &Cross;
+operator.\u2A53.infix = lspace:4 rspace:4 direction:vertical # &And;
+operator.\u2A54.infix = lspace:4 rspace:4 direction:vertical # &Or;
+operator.\u2A75.infix = lspace:5 rspace:5 # &Equal;
+operator.\u2A7D.infix = lspace:5 rspace:5 # &LessSlantEqual;
+operator.\u2A7D\u0338.infix = lspace:5 rspace:5 # &NotLessSlantEqual;
+operator.\u2A7E.infix = lspace:5 rspace:5 # &GreaterSlantEqual;
+operator.\u2A7E\u0338.infix = lspace:5 rspace:5 # &NotGreaterSlantEqual;
+operator.\u2A85.infix = lspace:5 rspace:5 # &lessapprox;
+operator.\u2A86.infix = lspace:5 rspace:5 # &gtrapprox;
+operator.\u2A8B.infix = lspace:5 rspace:5 # &lesseqqgtr;
+operator.\u2A8C.infix = lspace:5 rspace:5 # &gtreqqless;
+operator.\u2AA1.infix = lspace:5 rspace:5 # &LessLess;
+operator.\u2AA1\u0338.infix = lspace:5 rspace:5 # &NotNestedLessLess;
+operator.\u2AA2.infix = lspace:5 rspace:5 # &GreaterGreater;
+operator.\u2AA2\u0338.infix = lspace:5 rspace:5 # &NotNestedGreaterGreater;
+operator.\u2AAF.infix = lspace:5 rspace:5 # &PrecedesEqual;
+operator.\u2AAF\u0338.infix = lspace:5 rspace:5 # &NotPrecedesEqual;
+operator.\u2AB0.infix = lspace:5 rspace:5 # &SucceedsEqual;
+operator.\u2AB0\u0338.infix = lspace:5 rspace:5 # &NotSucceedsEqual;
+operator.\u2AB3.infix = lspace:5 rspace:5 # &prE;
+operator.\u2AB4.infix = lspace:5 rspace:5 # &scE;
+operator.\u2AB7.infix = lspace:5 rspace:5 # &precapprox;
+operator.\u2AB8.infix = lspace:5 rspace:5 # &succapprox;
+operator.\u2AC5.infix = lspace:5 rspace:5 # &subseteqq;
+operator.\u2AC6.infix = lspace:5 rspace:5 # &supseteqq;
+operator.\u2AE4.infix = lspace:5 rspace:5 # &DoubleLeftTee;
+
+# Entries below are not part of the official MathML dictionary
+
+operator.\u0021.prefix = lspace:0 rspace:5 # !
+operator.\u0026.infix = lspace:5 rspace:5 # &amp;
+operator.\u0026.prefix = lspace:0 rspace:5 # &amp;
+operator.\u002B\u002B.prefix = lspace:0 rspace:2 # ++
+operator.\u002D\u002D.prefix = lspace:0 rspace:2 # --
+operator.\u003B.postfix = lspace:0 rspace:0 separator # ;
+operator.\u003C\u20D2.infix = lspace:5 rspace:5 # &nvlt;
+operator.\u003E\u20D2.infix = lspace:5 rspace:5 # &nvgt;
+operator.\u006C\u0069\u006D.prefix = lspace:0 rspace:3 movablelimits # lim
+operator.\u006D\u0061\u0078.prefix = lspace:0 rspace:3 movablelimits # max
+operator.\u006D\u0069\u006E.prefix = lspace:0 rspace:3 movablelimits # min
+operator.\u007E.infix = lspace:2 rspace:2 stretchy direction:horizontal # ~
+operator.\u0332.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &UnderBar;
+operator.\u2016.infix = lspace:5 rspace:5 stretchy direction:vertical # &Vert; &Verbar;
+operator.\u20D0.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20D0;
+operator.\u20D1.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20D1;
+operator.\u20D6.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20D6;
+operator.\u20D7.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20D7;
+operator.\u20E1.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20E1;
+operator.\u2190\u200B.infix = lspace:5 rspace:5 # &ShortLeftArrow;
+operator.\u2191\u200B.infix = lspace:2 rspace:2 # &ShortUpArrow;
+operator.\u2192\u200B.infix = lspace:5 rspace:5 # &ShortRightArrow;
+operator.\u2193\u200B.infix = lspace:2 rspace:2 # &ShortDownArrow;
+operator.\u2210.infix = lspace:3 rspace:3 direction:vertical # &Coproduct;
+operator.\u2223.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &VerticalBar;
+operator.\u2223.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &VerticalBar;
+operator.\u2225.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &DoubleVerticalBar;
+operator.\u2225.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &DoubleVerticalBar;
+operator.\u2282\u020D2.infix = lspace:5 rspace:5 # &NotSubset;
+operator.\u2283\u020D2.infix = lspace:5 rspace:5 # &NotSuperset;
+operator.\u228E.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &UnionPlus;
+operator.\u2295.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &CirclePlus;
+operator.\u2296.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &CircleMinus;
+operator.\u2297.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &CircleTimes;
+operator.\u2299.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &CircleDot;
+operator.\u22C0.infix = lspace:3 rspace:3 direction:vertical # &Wedge;
+operator.\u22C1.infix = lspace:3 rspace:3 direction:vertical # &Vee;
+operator.\u23B0.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &lmoustache; &lmoust;
+operator.\u23B1.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &rmoustache; &rmoust;
+operator.\u2500.infix = lspace:0 rspace:0 stretchy direction:horizontal # &HorizontalLine;
+operator.\u25A1.prefix = lspace:0 rspace:2 # &Square;
+operator.\u2606.infix = lspace:3 rspace:3 # &star;
+operator.\u2AC5\u0338.infix = lspace:5 rspace:5 # &nsubseteqq;
+operator.\u2AC6\u0338.infix = lspace:5 rspace:5 # &nsubseteqq;
+operator.\u2AEC.prefix = lspace:0 rspace:5 # &Not;
+operator.\uFE35.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &OverParenthesis; (MathML 2.0)
+operator.\uFE36.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &UnderParenthesis; (MathML 2.0)
+operator.\uFE37.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &OverBrace; (MathML 2.0)
+operator.\uFE38.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &UnderBrace; (MathML 2.0)
 
 
 ##################################################################################
 # DOCUMENTATION ON HOW TO SETUP THE PROPERTY FILE ASSOCIATED TO EACH FONT
 # More fonts can be supported for stretchy characters by setting up mathfont
 # property files as described below.
 #
 # Each font should have its set of glyph data. For example, the glyph data for
